src/components/cw20-token/services/cw20-token.service.ts

```diff
@@ -112,29 +96,14 @@ export class Cw20TokenService {
         address
       }`;
 
-    const isSearchByContractAddress =
```

Comment:
sao lại bỏ chỗ này đi a

Reply:
ở BE sẽ lấy all data từ horoscope về việc filter hoặc search FE sẽ xử lí em.